cms/src/test/java/uk/nhs/digital/apispecs/swagger/SwaggerCodeGenOpenApiSpecificationJsonToHtmlConverterTest.java

```diff
@@ -387,17 +387,36 @@ public void rendersOperationIds_normalisedToContiguousString() {
         // given
         final String specificationJson = from("oasV3_operationIdsInVariousFormats.json");
 
-        final String expectedSpecHtml = from("oasV3_operationIdsInVariousFormats.html");
-
         // when
         final String actualSpecHtml = swaggerCodeGenApiSpecHtmlProvider.htmlFrom(specificationJson);
 
         // then
+        // @formatter:off
         assertThat(
             "OperationID rendered as it is on the specification - Camel case and snake case tested.",
-            ignoringUuids(ignoringWhiteSpacesIn(actualSpecHtml)),
-            is(ignoringUuids(ignoringWhiteSpacesIn(expectedSpecHtml)))
+            actualSpecHtml,
+            stringContainsInOrder(
```

Comment:
great work making this test less fragile and tightening the assertion logic

Reply:
Thanks, although this sword has two edges, as I have learned the hard way.

When HTML result is large and the test fails, this approach actually makes it more difficult to spot what exactly the mismatch is. I was one third into streamlining `rendersAllSupportedRequestAndResponseParameters` this way and had to back out when I realised this.

This approach seems fine for smaller payloads but has some cost with larger ones. I may give `rendersAllSupportedRequestAndResponseParameters` another go when I have more time to think about it.